docker/tasks/main.yml

```diff
@@ -25,10 +25,10 @@
     cache_valid_time: 600
 
 - user:
-     name: conductr
+     name: conductr-agent
 
 - name: Non-root access
-  command: usermod -a -G docker conductr
+  command: usermod -a -G docker conductr-agent
```

Comment:
Correct. The conductr user does not execute docker commands in 2.0.0/master, conductr-agent does